dolphinscheduler-task-plugin/dolphinscheduler-task-http/src/main/java/org/apache/dolphinscheduler/plugin/task/http/HttpParameters.java

```diff
@@ -66,83 +56,10 @@ public class HttpParameters extends AbstractParameters {
      */
     private int connectTimeout;
 
-    /**
-     * Socket Timeout
-     * Unit: ms
-     */
-    private int socketTimeout;
-
     @Override
     public boolean checkParameters() {
-        return StringUtils.isNotEmpty(url);
-    }
-
-    @Override
-    public List<ResourceInfo> getResourceFilesList() {
-        return new ArrayList<>();
-    }
-
-    public String getUrl() {
-        return url;
-    }
-
-    public void setUrl(String url) {
-        this.url = url;
-    }
-
-    public HttpMethod getHttpMethod() {
-        return httpMethod;
-    }
-
-    public void setHttpMethod(HttpMethod httpMethod) {
-        this.httpMethod = httpMethod;
-    }
-
-    public List<HttpProperty> getHttpParams() {
-        return httpParams;
-    }
-
-    public void setHttpParams(List<HttpProperty> httpParams) {
-        this.httpParams = httpParams;
-    }
-
-    public HttpCheckCondition getHttpCheckCondition() {
-        return httpCheckCondition;
-    }
-
-    public void setHttpCheckCondition(HttpCheckCondition httpCheckCondition) {
-        this.httpCheckCondition = httpCheckCondition;
-    }
-
-    public String getCondition() {
-        return condition;
+        return StringUtils.isNotEmpty(url) && httpRequestMethod != null
+                && connectTimeout > 0;
```

Comment:
Don't need to check connect timeout here, since you will use default value.

Reply:
If the user does not use the web-ui, but uses the api to make parameter calls, there may be an exception here.